src/Simplify.elm

```diff
@@ -10359,26 +10352,52 @@ Use `returnsArgError` with the given last arg as `arg` when the last arg is alre
 alwaysReturnsLastArgError :
     String
     ->
-        { lastArgRepresents : String
-        , lastArg : Maybe (Node Expression)
+        { remainingArgCount : Int
+        , remainingArgs : List (Node Expression)
```

Comment:
why not use `argCount` and `argsAfterFirst` from checkInfo to determine the level of currying?

Reply:
Because it's not always that number of arguments.
Example (https://github.com/jfmengels/elm-review-simplify/pull/207/files#diff-6239bccafe2b75c7e560542732acf25d142d08608e1e0f0ffe9b53c09cfb1e0eR4837):
```elm
String.replace n n
```
Here we're only expecting 1 more argument because we already have one.

Reply:
`argsAfterFirst` tells you exactly how many are actually present, right?
So together with `argCount` we already have all the info we need to determine both the last arg and how many args before which aren't present need to be ignored

Reply:
Hmm... yeah you're right, they're equivalent in practice :+1: